py/dml/dmlparse.py

```diff
@@ -2670,7 +2670,9 @@ def reserved_(t):
 @prod_dml14
 @lex.TOKEN(ident_rule('ident', ['_']))
 def ident_underscore(t):
-    if compat.discard_ref_shadowing not in dml.globals.enabled_compat:
+    if (compat.discard_ref_shadowing not in dml.globals.enabled_compat
+        # forgive the `param _ auto` declaration
+        and not site(t).filename().endswith('dml-builtins.dml'):
```

Comment:
we don't have a `param _ auto` declarqtion!

Reply:
also do please actually go through the effort to build and test your changes before pushing them to to another's branch. That's just rude.

Reply:
oh, sorry, I was so sure we had that I didn't even bother to check. So the top commit is rubbish, now removed.